src/amb_kf.c

```diff
@@ -853,11 +857,30 @@ void nkf_state_projection(nkf_t *kf,
   /* NOTE: IT DOESN'T UPDATE THE OBSERVATION OR TRANSITION MATRICES, JUST THE STATE. */
 }
 
+/** Add new sats the the KF
+ * Given some space Z = X x Y, and some state mean/cov on X,
+ * we construct a state mean/cov on Z by doing the inclusion of X
+ * in Z and making initial estimates for the state of the Y elements
+ * of Z. Here, X is the space of old satellite DDs, and Y is new sats.
+ *
+ * \param kf                    The KF struct to be updated
+ * \param num_old_non_ref_sats  The old number of DDs (dimension of X)
+ * \param num_new_non_ref_sats  The new number of DDs (dimension of Z)
+ * \param ndx_of_old_sat_in_new The indices of the old sats in the new full
+ *                              space. For example, if the first (ndx=0) element
+ *                              of the state vector in X was for PRN 3 and
+ *                              PRN 3 is the 4th (ndx=3) element of the state
+ *                              vector in Z, then ndx_of_old_sat_in_new[0] = 3.
+ * \param estimates             Rough estimates of the ambiguities (in Z).
```

Comment:
Again drop rough and consider expanding on the fact that this should be length `num_new_non_ref_sats`, and is used to initialise the starting value of the state mean for the new satellites.